src/controllers/controlpickermenu.cpp

```diff
@@ -89,6 +89,8 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
     addDeckControl("beats_translate_match_alignment", tr("Adjust Beatgrid - Match Alignment"),
                    tr("Adjust beatgrid to match another playing deck."), bpmMenu);
     addDeckAndSamplerControl("quantize", tr("Quantize Mode"), tr("Toggle quantize mode"), bpmMenu);
+    addDeckAndSamplerControl("beats_set_bar_beat", tr("Set Bar Beat"),
```

Comment:
I know bars and beats, but now "bar beats", Can we rename it to just "bar" or "down beat"

Reply:
"downbeat" seems to be common: https://en.wikipedia.org/wiki/Beat_(music)#Downbeat_and_upbeat -> `beats_set_downbeat` + "Set first beat of a bar"